README.md

```diff
@@ -41,6 +41,28 @@ See the following examples for more detail on bundling OpenLayers with your appl
  * Using [Parcel](https://github.com/openlayers/ol-parcel)
  * Using [Browserify](https://github.com/openlayers/ol-browserify)
 
+## TypeScript and VS Code IntelliSense support
+
+The `ol` package contains a `src/` folder with JSDoc annotated sources. TypeScript can get type definitions from these sources with a `tsconfig.json` like this:
+```js
+{
+  "compilerOptions": {
+    // Enable JavaScript support
+    "allowJs": true,
+    // Point to the JSDoc typed sources when using modules from the ol package
+    "baseUrl": "./",
+    "paths": {
+      "ol": ["node_modules/ol/src"],
+      "ol/*": ["node_modules/ol/src/*"]
```

Comment:
I think the final `ol` is missing:
```suggestion
      "ol/*": ["node_modules/ol/src/ol/*"]
```
But I cannot confirm without havinf a npm package to test

Reply:
No, the folder structure in the package is `src/`, and below that everything that is under `src/ol/` in the repo. But I can change that structure if it is misleading.

BTW, you can always test with

    npm run build-package
    cd build/ol
    npm link

in the OpenLayers repo, then go to the project that has `ol` as dependency, and there

    npm link ol